OpenTabletDriver.Configurations/Configurations/Wacom/DTK-2200.json

```
@@ -0,0 +1,40 @@
+{
+  "Name": "Wacom Cintiq 22HD (DTK2200)",
```

I fixed the name for this in #3475, so you should mirror that here.
```suggestion
  "Name": "Wacom Cintiq 22HD (DTK-2200)",
```

---

Oops, thanks.